documentation/Windows-Installation-Guide.md

```diff
@@ -0,0 +1,399 @@
+**PostGis** and other **JDBC** and **ODBC** drivers. Click **Finish**  
+![Pg installation complete](https://user-images.githubusercontent.com/19670372/33494846-7be6c9f2-d6bc-11e7-874c-2f10d1a84e07.png)
+
+### Selecting Drivers and Spatial Extensions
+
+- Stack builder installation dialog will pop up, choose the **Postgresql** version you just downloaded in the 
+drop down box, Click **Next**
+- On below screen, expand database drivers and choose, **pgJDBC** and **psqlODBC** as shown in the screenshot below. 
+(make sure for **psqlODBC** you choose the  right architecture). Also expand the **Spatial Extensions** section 
+and choose **PostGIS** option. Click **Next**
+![Stack builder](https://user-images.githubusercontent.com/19670372/33494841-7b6dcf5c-d6bc-11e7-8b8b-6074c48097b3.png)
+- Leave the download directory as default and click **Next** and click **Next** again
+
+##### JDBC Installation
+- Click **Next** on the below screen and **JDBC** installation will begin
+![Jdbc installation](https://user-images.githubusercontent.com/19670372/33494838-7b0e7480-d6bc-11e7-97f2-2d55d7bf656a.png)
+
+- Once the installation is done you would see a screen like this, but the installation is only completed for 
+one driver, we still have got 2 more drivers to install as we have selected 3 drivers on Stack Builder. 
+Click **Finish**
+
+##### ODBC Installation
+
+- Once you are done with the **JDBC** setup, next setup will start which is **ODBC**, click **Next**
+- Leave the default installation directory and click **Next** and Click **Next** again and installation will begin
+![ODBC Installation](https://user-images.githubusercontent.com/19670372/33494835-7ac5598a-d6bc-11e7-93f4-29e01690b677.png)
+- Once the installation is done, click **Finish**.
+
+##### Postgis Installation
+
+-  Now the third installation will start which is **PostGis**, accept the terms and conditions by clicking **I Agree**  
+![Postgis agreement](https://user-images.githubusercontent.com/19670372/33494832-7a93ce24-d6bc-11e7-993b-d8e99c2bdd23.png)
+- Click **Next** on **Choose Components** screen and make sure **PostGIS** is selected.
+- Leave the installation directory to default on **Choose Install Location** screen and Click **Next**
+- Click **Yes** on all the below shown dialogs  
+![postgis gdal data](https://user-images.githubusercontent.com/19670372/33494829-7a455e38-d6bc-11e7-8c98-51c8a91c03d6.png)
+![postgis rasters](https://user-images.githubusercontent.com/19670372/33494827-7a2c323c-d6bc-11e7-8e67-0a9f5ed5d5ef.png)
+![postgis rasters 2](https://user-images.githubusercontent.com/19670372/33494826-79fad25a-d6bc-11e7-8b63-1a35044e23d4.png)
+- Click **Close** when the installation is finished
+- Click **Finish** on **Stack Builder Installation Completed** screen
+
+###### Note: We still have to set the Environment Variable path, as psql command will still not work on Command Prompt
+
+##### Set Environment Variable
```

Comment:
This is a comment in general for the `Set Environment Variable`:
The user needs to set it for all so this could be just one step for all of them. You could link it in each section in case the user does not go through all the guide in one og.

Reply:
The idea here is to re-iterate to user to set environment variable as there may be a case where user have java installed but not gradle, thus if we take out this comment user might would just miss the step for setting up env. If user would follow this guide as end to end then i think it makes sense to have set env variable as a separate section

Reply:
The link to sections will help here, so it's ok.